src/device/pointer.jl

```diff
@@ -4,19 +4,19 @@
 # Address spaces
 #
 
+export AS, addrspace
+
 abstract type AddressSpace end
 
 module AS
 
-using AMDGPUnative
-import AMDGPUnative: AddressSpace
+import ..AddressSpace
 
 struct Generic  <: AddressSpace end
 struct Global   <: AddressSpace end
-struct Region   <: AddressSpace end
```

Comment:
Didn't these AS have different names because they were derived from HSA?

Reply:
Yes, that's a good point. I'll switch these back.